code/datums/mind/antag.dm

```diff
@@ -164,7 +169,7 @@
 	new_uplink.uplink_handler.assigned_role = traitor_mob.mind.assigned_role.title
 	new_uplink.uplink_handler.assigned_species = traitor_mob.dna.species.id
 	if(uplink_loc == R)
-		unlock_text = "Your Uplink is cunningly disguised as your [R.name]. Simply dial the frequency [format_frequency(new_uplink.unlock_code)] to unlock its hidden features."
+		unlock_text = "Your Uplink is cunningly disguised as your [R.name]. Simply speak \"[new_uplink.unlock_code]\" into frequency :d to unlock its hidden features."
```

Comment:
Is this information in the antag menu somewhere? How will you know if you miss this message?

Reply:
```
/datum/antagonist/traitor/ui_static_data(mob/user)
	var/datum/component/uplink/uplink = uplink_ref?.resolve()
	var/list/data = list()
...
		data["uplink_unlock_info"] = uplink.unlock_text``` yep, this unlock text is displayed under the antag menu

Reply:
Ok cool

Reply:
and it should also be under your ic notes, since it mentions :d in there too